index.js

```diff
@@ -151,6 +151,19 @@ MetaInspector.prototype.getMetaDescription = function()
 	return this;
 }
 
+MetaInspector.prototype.elemContainsTag = function(elem,tag) {
+	if (elem.name && elem.name === tag ) {
```

Comment:
I think we should use `elem.tagName` as `elem.name` appears to be limited in the types of elements it applies to (see note on https://developer.mozilla.org/en-US/docs/Web/API/Element/name). Also it might be clearer if the `tag` parameter were renamed to `tagName` to indicate it's a name and not a tag object.